modules/admin/con-rhdh-deployment-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+// assembly-rhdh-operator-deployment-config
+
+[id="con-rhdh-deployment-config_{context}"]
+= {product} deployment configuration
```

Comment:
This is a brand new mmethod to configure product.

Consider:

* Including rather in assemblies/assembly-add-custom-app-file-openshift.adoc

* Transforming into a procedure with Title: Configuring product on OCP using the operator

* Adding an example that specifically configures the  "app-config-rhdh.yaml" or "app-config.yaml" file.

* Make this procedure the first one in assembly-add-custom-app-file-openshift.adoc

* This new procedure should replace the Adding a custom application configuration file procedure (that was actually a workaround to avoid the file to be rewritten by the operator).

Reply:
Does this apply to only OpenShift? Or does it apply to all platforms?

Reply:
This is about configuring how the RHDH Deployment should look like at the infra level (like adding new labels, a new sidecar container, adding volumes, using a different storage class, ...).
`assemblies/assembly-add-custom-app-file-openshift.adoc` is mostly about the application configuration.
So to me, they are quite unrelated.